nwb_conversion_tools/datainterfaces/axonadatainterface.py

```diff
@@ -0,0 +1,316 @@
+"""Authors: Steffen Buergers"""
+import os
+import re
+import datetime
+import contextlib
+import mmap
+import numpy as np
+
+import spikeextractors as se
+
+from pynwb import NWBFile
+from pynwb.behavior import Position, SpatialSeries
+
+from ..basedatainterface import BaseDataInterface
+from ..baserecordingextractorinterface import (
+    BaseRecordingExtractorInterface
+)
+
+from spikeextractors.extractors.nwbextractors.nwbextractors import check_module
+
+
+# Helper functions for AxonaRecordingExtractorInterface
+def parse_generic_header(filename, params):
+    """
+    Given a binary file with phrases and line breaks, enters the
+    first word of a phrase as dictionary key and the following
+    string (without linebreaks) as value. Returns the dictionary.
+
+    INPUT
+    filename (str): .set file path and name.
+    params (list or set): parameter names to search for.
+
+    OUTPUT
+    header (dict): dictionary with keys being the parameters that
+                   were found & values being strings of the data.
+
+    EXAMPLE
+    parse_generic_header('myset_file.set', ['experimenter', 'trial_time'])
+    """
+    header = {}
```

Comment:
```suggestion
    header = dict()
```
Please use dictionary literals whenever possible. The time and place to use `{}` is under compression with values that can't easily be mapped as strings, which is somewhat rare.

Reply:
Ok!